### ISSUE TITLE
Improve CV efficiency for Lasso, Elastic Net, etc.

### ISSUE THREAD

**Issue by ryanbanderson** [USGS-Astrogeology/PySAT_Point_Spectra_GUI#166]
The sklearn documentation [talks ](http://scikit-learn.org/stable/modules/grid_search.html#model-specific-cross-validation)about taking advantage of the way these algorithms work to essentially get the performance at a bunch of values of alpha all at once. These methods currently are very slow in our current cross validation implementation, so we should look closer at the sklearn CV code for these methods and see if we can do better.

**Comment by ryanbanderson** [comment]
Also, for these methods we should throw a warning when the lowest RMSECV as a function of alpha is at the edge of the range of alphas (indicating that CV should be re-run with a wider range to find the true minimum)

**Comment by ryanbanderson** [comment]
Done for Lasso and Elastic Net, so I'm closing this issue. Ridge is pretty fast anyway so not going to worry about it.
